Lib/profiling/sampling/__init__.py

```diff
@@ -7,6 +7,7 @@
 from .collector import Collector
 from .pstats_collector import PstatsCollector
 from .stack_collector import CollapsedStackCollector
+from .gecko_collector import GeckoCollector
```

Comment:
ℹ️ **Testing**

The addition of `GeckoCollector` to the imports suggests a new feature or functionality. Ensure that the `GeckoCollector` class is well-documented and tested. Consider adding unit tests to cover its functionality if not already present.